scope.py

```diff
@@ -750,6 +750,46 @@ def test(self):
             # clean up after thyself
             (path_mock / dataset).unlink()
 
+    def get_highscoring_objects(otype='vnv',condition="$or",
```

Comment:
Maybe this stuff belongs in its own file in /scope? like /scope/fritz.py or something?

Reply:
@janvanroestel ?